scripts/npm/install-dependencies.sh

```diff
@@ -10,7 +10,7 @@ if diff -q $SHRINKWRAP_FILE $SHRINKWRAP_CACHED_FILE; then
 else
   echo 'Blowing away node_modules and reinstalling npm dependencies...'
   rm -rf node_modules
-  npm install
+  npm install --registry http://registry.npmjs.org/
```

Comment:
I guess this has to do with your `grunt package` issues, but it definitely does not belong in this PR :)